src/modules/mocked-urls/mocked-urls.repository.js

```diff
@@ -0,0 +1,16 @@
+const fs = require("fs");
+
+const MOCKED_URLS = "./static/mocked-urls.json";
```

Comment:
The default could be a `mocked-urls.json` on the same directory, think of using this project as an executable.

I would suggest:
```suggestion
const mockedUrlsPath = process.env.MOCKED_URLS_PATH || "mocked-urls.json";
```

Reply:
Good one. I did it like:   
```js
const mockedUrlsPath = process.env.MOCKED_URLS_PATH || "mocked-urls.json";

const getMockedUrls = () => {
  return JSON.parse(fs.readFileSync(join(__dirname, mockedUrlsPath), "utf8"));
};

const saveMockedUrls = ({content}) => {
  fs.writeFileSync(join(__dirname, mockedUrlsPath), JSON.stringify(content));
};
```